src/components/pages/AuthPages/Login.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import PageLayout from '<templates>/PageLayout/PageLayout';
+import AuthForm from '<organisms>/AuthForm/AuthForm';
+import withCheckAuth from '<HOC>/withCheckAuth';
+import { loginData, loginRules } from './loginItems';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import ImageUrl from '<image>/home.png';
+import { signin } from './navItems';
+import headerMapper from '<helpers>/headerMapper';
+
+const navItems = headerMapper(signin);
+
+const Login = (props) => {
+  return (
+    <PageLayout navItems={navItems}>
+      <FlexContainer>
+        <FlexContainer flexDirection='row' height='80%'>
```

Comment:
i think you should use the already existing image component since it is being reused.